src/applications/claims-status/containers/OverviewPage.jsx

```diff
@@ -30,7 +26,7 @@ const formatDate = buildDateFormatter(DATE_FORMATS.LONG_DATE);
 const STATUSES = getStatusMap();
 
 const getPhaseFromStatus = latestStatus =>
-  [...STATUSES.keys()].indexOf(latestStatus) + 1;
+  [...STATUSES.keys()].indexOf(latestStatus.toUpperCase()) + 1;
```

Comment:
vets-api-mockdata is using `Complete` for the status instead of `COMPLETE` so added `toUpperCase()` to account for outdated mockdata